codelabs/README.md

```diff
@@ -0,0 +1,8 @@
+To build the codelabs you need to install Go and the codelabs claat tool:
+
+- Install go: https://golang.org/doc/install
+- Setup go in your bash profile: `export PATH=$PATH:$HOME/go/bin`
+- Install claat with `go get github.com/googlecodelabs/tools/claat`
+- Run yarn codelabs:build
+
+See https://medium.com/@zarinlo/publish-technical-tutorials-in-google-codelab-format-b07ef76972cd for a full guide, and https://github.com/googlecodelabs/tools for the CLI tool itself.
```

Comment:
we wanna use Google code labs? I find it rather complicated for what it does? we should be able to reproduce it with web components right?

also nice would be to only store the "source" in git and generate the output when netlify is building the page (probably not that easy with Google code labs)
otherwise contributing will be really tough 😥

Reply:
It's just a single command line, so it's not really complicated other than that you need to install Go. 

Building our own solution is just more to maintain again. However we could use the codelab web components directly, and we could just read and parse the markdown in the browser it's not that complex.

The building should happen in netlify for sure, where is the configuration for this stored?